sql_scripts/2_states.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE IF NOT EXISTS states
+(
+    id SMALLSERIAL PRIMARY KEY,
```

Comment:
Consider switching to INTEGER and declare id during INSERT, cause we don't want to insert same values in table during each deployment